src/testFixtures/kotlin/com/example/delivery/OrderFixture.kt

```diff
@@ -0,0 +1,66 @@
+package com.example.delivery
+
+import com.example.delivery.domain.DomainOrder
+import com.example.delivery.dto.request.CreateOrderDTO
+import com.example.delivery.dto.response.OrderDTO
+import com.example.delivery.dto.response.ShipmentDetailsDTO
+import com.example.delivery.mongo.MongoOrder
+import org.bson.types.ObjectId
+import java.math.BigDecimal
+
+object OrderFixture {
+    val order = MongoOrder(
+        id = ObjectId("123456789011121314151617"),
```

Comment:
it is good practice to generate random values for test data. Follow it in next fixtures, please